### PR TITLE
Add project YieldYak

### PR DESCRIPTION
**Pull request by Jack-ZooFiLabs**
Smart contracts of three github repo:
1. yieldyak/smart-contracts
2. yieldyak/governance-contracts
3. yieldyak/farm-contracts

### CODE REVIEW

File: projects/yield-yak.json [hunk 1]
```diff
@@ -0,0 +1,3401 @@
+[
+  {
+    "name": "BAG-BAG",
+    "address": "0xf487044eD85F2d47A8eAD6b86c834976B8c31736",
+    "chain": "avalanche",
+    "version": "v1",
+    "type": "type",
```
Comment (rbval) [inline projects/yield-yak.json:7]: These should be should be either `core` or `governance`, for example:

`"type": "core"`

File: projects/yield-yak.json [hunk 2]
```diff
@@ -0,0 +1,3401 @@
...
+    "address": "0xdDAaAD7366B455AfF8E7c82940C43CEB5829B604",
+    "chain": "avalanche",
+    "version": "v1",
+    "type": "type",
+    "description": "A mYak is one-million-th Yak",
+    "explorers": [
+      "https://cchain.explorer.avax.network/address/0xdDAaAD7366B455AfF8E7c82940C43CEB5829B604"
+    ]
+  }
+]
```
Comment (rbval) [inline projects/yield-yak.json:3401]: Nitpick: add a newline here.

**Review by rbval — CHANGES_REQUESTED** [review]
Few small changes and we're good to go:
1. Add `core` or `governance` to `type`
2. Add newline to the end

**Comment by Jack-ZooFiLabs** [comment]
Done. Please take a look.

**Comment by fnery** [comment]
Thanks @Jack-ZooFiLabs. Ideally we'd appreciate the following fixes:
- fix versions that look like `vDAI`, `vETH`, `vLINK` (we expect something like `v1`, `v2`, etc...)
- could we have a bit more specific governance contract descriptions?
- checksummed contracts in explorer URLs -> lowercase contracts

**Comment by Jack-ZooFiLabs** [comment]
Ok. Fixed a bug in the parser that gives "vDAI" etc. 

Re governance contract description, I'm not most familiar with it. I'll ask my teammate to send another pull request to update it. This is a bulk import done with a scanner. :)

Used lowercase contracts address.

**Review by rbval — APPROVED** [review]

**Pull request merged**